lib/zwave/ZwaveDevice.js

```diff
@@ -248,11 +258,17 @@ class ZwaveDevice extends MeshDevice {
 				},
 				'Configuration Value': parsedValue || value,
 			}, (err, result) => {
+				const parsedBufValue = parsedValue.toString('hex').toUpperCase();
+
+				let parsedDecValue;
+				if (!options.hasOwnProperty('signed') || options.signed === true) parsedDecValue = parsedValue.readIntBE(0, options.size);
```

Comment:
These reads are a bit error prone, I think they should be try/catched. By the way, the code executed in the callback here can also be performed before configuration setting is applied, it does not depend on the result of the `CONFIGURATION_SET`.

Reply:
not sure what you are trying to say with "executed before" as the code for showing what changed was already there, only expanded on it.

Reply:
Exactly that's why it can happen 'synchronously' before `commandClassConfiguration.CONFIGURATION_SET()` is called. It doesn't really matter, but if possible I prefer synchronous code (since there is no gain).